go.work

```diff
@@ -0,0 +1,9 @@
+go 1.23.6
+
+use (
+	./lib/openslides-go
+	./openslides-autoupdate-service
```

Comment:
Can we change this `use` to reduce the dependencies between the go serviceres? I don't like the mount of all go-services into every container in the `docker-compose.dev.yml` above that is a result of this `use` block.

Reply:
IMHO for the `go.work` file this is okay. We could change the `go.work` files that are added to the containers to just contain the openslides-go module.

Reply:
If we get rid of the unnecessary "dependencies", thats the way.